app/scripts/components/home.js

```diff
@@ -110,6 +117,26 @@ class Home extends React.Component {
               </ul>
             </div>
           </section>
+          <section className='page_section'>
+            <div className='row'>
+              <div className='heading__wrapper--border'>
+                  <h2 className='heading--medium heading--shared-content--right'>Distribution Metrics</h2>
+              </div>
+              <ul>
+                {distStats.map(d => {
```

Comment:
I see that this block is used above for other numbers. Can we copy this render logic into a re-usable method on this component? Or a separate component if it recurs elsewhere

Reply:
8339f0a